app/collections/show.cjsx

```diff
@@ -34,27 +38,61 @@ CollectionPage = React.createClass
   componentWillUnmount: ->
     document.documentElement.classList.remove 'on-collection-page'
 
+  getBaseLink: ->
+    baseLink = "/"
+    if @props.project?
+      baseLink += "projects/#{@props.project.slug}"
+    baseLink
+
+  getOwnerLogin: ->
+    [owner_login, collection_name] = @props.collection.slug.split('/')
+    owner_login
+
+  getCollectionsLink: ->
+    baseLink = @getBaseLink()
+    [owner_login, collection_name] = @props.collection.slug.split('/')
+    if @props.collection.favorite
+      "#{baseLink}favorites/#{@getOwnerLogin()}"
+    else
+      "#{baseLink}collections/#{@getOwnerLogin()}"
+
+  getProfileLink: ->
+    baseLink = @getBaseLink()
+    "#{baseLink}users/#{@getOwnerLogin()}"
+
+  getCollectionsLinkMessageKey: ->
+    if @props.collection.favorite
+      "collectionPage.favoritesLink"
+    else
+      "collectionPage.collectionsLink"
+
   render: ->
     <PromiseRenderer promise={@props.collection.get('owner')}>{(owner) =>
-      baseLink = "/collections/#{@props.collection.slug}"
+      baseCollectionsLink = "#{@getBaseLink()}collections/#{@props.collection.slug}/"
```

Comment:
Sorry, this is where the missing slash is coming from.  Should be `baseCollectionsLink = "#{@getBaseLink()}/collections/#{@props.collection.slug}/"`